Cookbook/Technical-Documents/Localizations.md

```diff
@@ -0,0 +1,132 @@
+![Create file, if needed](Assets/Localizations-createFile.png)
+
+- Select the `.strings` file in the Project Navigator, review it in the File Inspector / Localization and select the language/locale you want.
+- In `Fastlane/Lanes/lokalise`, add the language/locale code (`en_US` for example) in the `langs` parameter for the desired target. `langs: 'es_US,en_US'` (**comma separated without space**). 
+
+### 2. Lokalise.co
+- If the project still doesn't exist, click on the `+` plus button beside other flags. Find the desired language on the list, and then add it.
+Note: someone else from your squad should have done this before.
+
+#### 2.1 Populate the files - pull from lokalise
+- Check [Lokalise pull guide](https://github.com/Babylonpartners/ios-playbook/blob/master/Cookbook/Technical-Documents/Lokalise.md).
+
+#### 2.2 Commit just your changes
+Stage just your changes (additions and editions) on git. Don't stage any additions nor change that you don't recognize. It's really nice to be proactive but not in this case. It might cause unexpected/premature changes. Yes, discard others's changes.
+
+## Target specific localizable - Lokalise project and `strings` files
+
+As the name suggests, it's a `.strings` file that contains texts intended to be present **only** in a specific app target.
+The values in this file will be priority over the main file value.
+<br><br>
+E.g: <br>
+In `Babylon/Supporting Files/Localization/Localizable.strings`:<br>
+```"consultation_gp_details_name" = "GP Name:";```<br>
+<br>
+And in `Babylon/Brand/Telus/TargetSpecificLocalizable.strings`:<br>
+```"consultation_gp_details_name" = "Doctor Name:";```<br>
+<br>
+When we run the target **Telus**, the value will be taken from the target specific file.
+`Doctor Name:`
+
+Whereas when we run **Babylon** the value will be the main one. 
+`GP Name:`
+
+This prioritization happens in the static method [`LocalizationCore.localizedName`](https://github.com/Babylonpartners/babylon-ios/blob/develop/SDK/BabylonCore/BabylonCore/Utilities/LocalizationCore.swift#L14) that basically tries first to get the value for such key from the target-specific file and if the result is still the key itself, then it falls back to the main value. 
+
+```swift
+public static func localizedName(key: String, bundle: Bundle = Bundle.main) -> String {
+    let targetLocalizationString = NSLocalizedString(key, tableName: "TargetSpecificLocalizable", comment: "")
+    let targetHasKey = targetLocalizationString != key
+    return targetHasKey ? targetLocalizationString : NSLocalizedString(key, bundle: bundle, comment: "")
+}
+```
+
+Such approach allows other targets to have their custom keys' values when, for some reason, the user uses the device in British English.
+
+Not only that, but also avoids us to have useless and risky repetitions like:<br>
+```"consultation_gp_details_name_babylon" = "GP Name:";```<br>
```

Comment:
Can't add suggestion because of the syntax 🔥 This should be:

\```
"consultation_gp_details_name_babylon" = "GP Name:";

(Note the code block starts before the string and not closed ,`<br>` removed)